package.json

```diff
@@ -99,6 +98,8 @@
     "handlebars": "4.7.7",
     "jest": "29.6.1",
     "jest-environment-jsdom": "29.6.1",
+    "jest-junit": "16.0.0",
```

Comment:
What are these packages for

Reply:
Which packages? jest-junit is used by the junit reporter. It was being provided by @storybook/test-runner as an implicit dependency, but now I added as an explicit one.